### ISSUE TITLE
Fix Icon Champ on macOS Ventura

### ISSUE THREAD

**Issue by philipamour** [MacEnhance/Forum#25]
Icons randomly change to a blue folder.

I've purchased the license just to change icons on macOS but the dev seems to be ghosting everyone and abandoning your project – don't waste your money.

**Comment by jordanbelinsky** [comment]
The same happened for me. The app no longer works at all...

**Comment by durosity** [comment]
Yeah It does seem to be abandonware at this point, which is frustrating as it's still available for sale on their website, but no attempts to contact them has been successful.  Grr.

**Comment by jmassmann** [comment]
Fix from another thread: remove the app under "App Management" in the privacy settings, added it again, restarted. Since then it works under Ventura for now.

**Comment by durosity** [comment]
> Fix from another thread: remove the app under "App Management" in the privacy settings, added it again, restarted. Since then it works under Ventura for now.

Alas I've tried this a few times and each time it soon reverts back to its prior behaviour.
